.github/workflows/aks.yml

```diff
@@ -36,38 +45,46 @@ jobs:
           go-version: ${{ env.SETUP_GO_VERSION }}
       - name: Setup Ginkgo Test Framework
         run: go install github.com/onsi/ginkgo/ginkgo@v1.16.2
-      - name: Install Dependencies
-        run: |
-          choco install kubernetes-cli eksctl
 
-      - name: Configure AWS credentials
+      - name: Configure AWS credentials for Route53
```

Comment:
Question: So while this is an MS/Azure Cluster, the DNS is done via Amazon ?!

Reply:
Guess that's fair for the test env - no need to test az-cli for DNS resources, when we implemented route53 in the first place. We don't provide code or even (provider-)specific docs on DNS setup - so thers's also nothing which could fail for us with Azure DNS. We do only need the "System-DOMAIN" and "Traefik-Ext-IP" - if there's something wrong with that, it would also fail with route53.

Reply:
The Route53 code was already there and we have our zones on AWS. Zone updates/DNS setup are not part of the test scenario, so I use the same zone.

Reply:
Please note that my question is not to imply that I am against this. I was simply confused to see Amazon things in a flow for MS/Azure. A small note in a comment that this mix is intentional would be enough to unconfuse future me reading it again. And hopefully other maintainers.

Reply:
This https://github.com/epinio/epinio/pull/729/files#diff-efbe2e406ecf70a5533ff28e4d18451538484832f80130758de3a649d8d03dbbR35 was not enough I guess ;)

Ok, will comment the workflow.

Reply:
Different location I only became aware of later, when reaching it in the PR. I prefer local explanations. Also ok are references to other relevant places and note of explanation there, although in second place after local explanations.